public/sass/components/_panel_header.scss

```diff
@@ -10,8 +10,10 @@ $panel-header-no-title-zindex: 1;
 .panel-container--no-title {
   .panel-header {
     position: absolute;
-    width: 100%;
+    width: calc(100% - 100px); // allow interaction in the corners
+    margin-left: 50px;
```

Comment:
i usually prefer `left`, though it does require `position: relative` or `position: absolute` (which is present here)